docs/AllBridgeFacet.md

```diff
@@ -0,0 +1,82 @@
+# AllBridge Bridge Facet
+
+## How it works
+
+The AllBridge bridge facet works by forwarding calls to the AllBridge core contract on the source chain. It is possible to find the core contract addresses and token pools by curling [token-info endpoint](https://core.api.allbridgecoreapi.net/token-info).
+
+One feature that makes AllBridge different from other bridges is that it uses pools on either side of the bridge. This allows them to no having to have canonical representation on the destination chain. Thus for EVM based chain this means that the pool needs to be approved to spend the bridged tokens.
+
+Underneath, AllBridge can use different message protocols to transfer tokens. These are represented in an enum like
+
+```
+/// @title AllBridge Messenger Protocol Enum
+enum MessengerProtocol {
+    None,
+    Allbridge,
+    Wormhole,
+    LayerZero
+}
+```
+
+```mermaid
+graph LR;
+    D{LiFiDiamond}-- DELEGATECALL -->C(AllBridgeFacet);
+    C(AllBridgeFacet) -- CALL --> C(AllBridge Core)
+```
+
+## Public Methods
+
+- `function startBridgeTokensViaBridgeFacet(BridgeData calldata _bridgeData, AllBridgeData calldata _allBridgeData)`
```

Comment:
The method name does not match with the facet

here: startBridgeTokensViaBridgeFacet()
facet: startBridgeTokensViaAllBridge() << correct

Reply:
updated